<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunManager</class>
 <widget class="QWidget" name="RunManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Config</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pbStart">
       <property name="toolTip">
        <string>Start Corry</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="leDir">
       <property name="toolTip">
        <string>working directory</string>
       </property>
       <property name="text">
        <string>~/tmp/corry</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="2">
      <widget class="QTableWidget" name="twEventLoaders">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;EventLoaders&lt;/p&gt;&lt;p&gt;File-name wildcard to be used for run-number&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <column>
        <property name="text">
         <string>Module</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Detector</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>File</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="leConfig">
       <property name="toolTip">
        <string>config file</string>
       </property>
       <property name="text">
        <string>config.cfg</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dir</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pbStop">
       <property name="toolTip">
        <string>Stop Corry</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="4">
      <widget class="QTextBrowser" name="tbCorryOutput"/>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="sbRun">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;run number&lt;/p&gt;&lt;p&gt;overrides file-names for event-loaders in config&lt;/p&gt;&lt;p&gt;when &amp;quot;-1&amp;quot; values in config file get used&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pbClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
